### DISCUSSION TITLE
Descrizioni in fattura

### DISCUSSION THREAD

**Discussion by GianmariaTS** [fattureincloud/api#258] [Q&A] [answered]
Ciao a tutti,

ho notato che sono cambiare le Note in merito alle esenzioni ad esempio:

versione V1:
Esente IVA ai sensi del regime del margine art.36 DL 41/95
Documento privo di valenza fiscale ai sensi dell'art. 21 Dpr 633/72. L'originale è disponibile all'indirizzo telematico da Lei fornito oppure nella Sua area riservata dell'Agenzia delle Entrate.

versione V2:
Esente IVA ai sensi del regime del margine art.36 DL 41/95
Copia del documento elettronico disponibile nella Sua area riservata dell'Agenzia delle Entrate

Vi risulta questo cambio di descrizioni?

**Comment by emajo** [comment]
Ciao @GianmariaTS non dovrebbero essere state fatte modifiche riguardanti queste informazioni, l'id passato nei campi `cod_iva` (v1) e `vat.id` (v2) è il medesimo?

Grazie,
Emanuele.

**Reply by GianmariaTS** [reply]
si uso gli stessi id ma trovo che in fase di visualizzazione i risultati sono diversi come nell'esempio, ti risulta?

**Reply by emajo** [reply]
Ciao @GianmariaTS, creando la fattura da web quale delle due diciture viene indicata?

**Reply by valmoz** [reply]
@GianmariaTS le serve ancora il nostro supporto?

Grazie
Mauro

**Comment by GianmariaTS** [comment, marked as answer]
mi scuso per il ritardo, non ho utilizzato la creazione della fattura online ma ho notato che dal giorno in cui è entrata in vigore la nuova versione effettivamente le descrizioni si sono aggiornate come nella versione 1 in particolare l'esempio:

Documento privo di valenza fiscale ai sensi dell'art. 21 Dpr 633/72. L'originale è disponibile all'indirizzo telematico da Lei fornito oppure nella Sua area riservata dell'Agenzia delle Entrate.

**Reply by valmoz** [reply]
Buongiorno @GianmariaTS ,
quindi possiamo chiudere la discussione?

Grazie
Mauro

**Reply by GianmariaTS** [reply]
si grazie a tutti per il supporto
